### DISCUSSION TITLE
how to access plotModel's Width & Height?

### DISCUSSION THREAD

**Discussion by CHANAHEE** [oxyplot/oxyplot#2022] [Q&A]
first, i use heatmap series. 
second, i added plotModel in plotView, 
but i won't get Width and Height for plotModel. 
when i debug, width and height value is zero. 

sorry for my english skills. please help me..

**Comment by VisualMelon** [comment]
The `Width` and `Height` etc. won't be populated until the plot is rendered. Unfortunately it's not trivial to 'dry-run' the rendering with the current rendering sequences.
